.github/workflows/test-rubikpi.yml

```
@@ -25,6 +25,19 @@ jobs:
               lsblk
               echo "${loopdev}"
         id: build_image
+
+      - name: Test using image file
+        uses: ./  # photonvision/photon-image-runner@HEAD
+        with:
+          image_url: ${{ steps.build_image.outputs.image }}
```

You need to define the offset in root_location, shrink_image, and boot_partition:

```suggestion
          image_url: ${{ steps.build_image.outputs.image }}
          root_location: "offset=569376768"
          shrink_image: "no"
          boot_partition: ""
```